### ISSUE TITLE
Modules still using BabyJSON

### ISSUE THREAD

**Issue by sivel** [ansible/ansible-modules-extras#430]
The following modules are still using BabyJSON to report errors/failures.  BabyJSON was support was removed in 1.8, so these errors will result in a parse error now.

cloud/lxc/lxc_container.py
cloud/misc/ovirt.py
cloud/misc/virt.py
notification/sns.py
system/firewalld.py

The proper way would be to use something similar to:

```
try:
    import foo
    HAS_FOO = True
except ImportError:
    HAS_FOO = False

[SNIP]

def main():
    module = AnsibleModule(...)

    if not HAS_FOO:
        module.fail_json(msg="The foo python module is required")
```

I don't have time right now to address or test these myself, but wanted to report this.

The one thing that may present problems is where a module is using a constant or value from the import when instantiating `AnsibleModule`

**Comment by robynbergeron** [comment]
@sivel Thanks for filing this. Way back when. :)

It looks like ovirt and sns are the two remaining modules using BabyJSON.

Copying @mjschultz (for sns.py) and @vincentvdk (ovirt.py) and @firstyear (who has also been doing some ovirt things recently) for their FYI.

**Comment by mjschultz** [comment]
https://github.com/ansible/ansible-modules-extras/pull/1045 should handle the SNS module.

**Comment by vincentvdk** [comment]
I'll have a look at it.

**Comment by nemobis** [comment]
Thanks for working on this: it's the reason I get

> failed=True msg='failed to connect to the firewalld daemon'
> OpenSSH_6.6.1, OpenSSL 1.0.1k-fips 8 Jan 2015

isn't it? Is there a workaround in ansible 1.9.4?

```
# dnf info firewalld
Installed Packages
Name        : firewalld
Arch        : noarch
Epoch       : 0
Version     : 0.3.14.2
Release     : 4.fc23
```

Update: nevermind, my fault; firewalld wasn't started yet due to another error. I guess it's rather https://github.com/ansible/ansible-modules-extras/issues/1112 :)

**Comment by mscherer** [comment]
So, it seems ovirt was fixed in 55dab7cd , so can I safely close this bug ?
